src/graphql/UserData.gql

```diff
@@ -0,0 +1,28 @@
+query($name: String!) { 
+  data: user(login: $name) {
+    name,
+    avatarUrl,
+    login,
+    repositories(first: 5, privacy: PUBLIC) {
+      totalCount,
+      pageInfo {
+        endCursor,
+        hasNextPage
+      }
+      nodes {
+        name,
+        object(expression: "master") {
+          ... on Commit {
+            history {
+              totalCount
+              nodes {
+                oid,
+                messageHeadline
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Policia das boas praticas: Sempre colocar uma linha adicional no fim do arquivo.

Uma das razões é que algumas ferramentas que leem ou trabalham com textos podem ter problemas para interpretar o final de um arquivo caso não tenha uma nova linha:

 [**_"A source file that is not empty shall end in a new-line character, which shall not be immediately preceded by a backslash character."_**](https://thoughtbot.com/blog/no-newline-at-end-of-file)